tests-functional/pages/base.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pypom import Page, Region
+from selenium.webdriver.common.by import By
+
+
+class Base(Page):
+
+    @property
+    def header(self):
+        return self.Header(self)
+
+    class Header(Region):
+
+        _root_locator = (By.ID, 'header')
+        _logo_locator = (By.ID, 'th-logo')
+
+        @property
+        def is_logo_displayed(self):
+            return self.is_element_displayed(*self._logo_locator)
```

Comment:
As far as I can tell, this isn't used. The base page is useful when you have common content across all pages, but if you don't need it you can either leave this empty (so you can use it in the future without having to change other page object parents) or you can remove it and make your other page objects inherit directly from `pypom.Page`.